src/03-control-flow.js

```diff
@@ -109,7 +137,18 @@ function aroundTheWorldGreeting(language) {}
  *  aroundTheWorldGreetingWithSwitch();
  *  //> "Hello World"
  */
-function aroundTheWorldGreetingWithSwitch(language) {}
+function aroundTheWorldGreetingWithSwitch(language) {
+  switch (language) {
+    case "german" :
+      return "Hallo Welt"
```

Comment:
As we said in class - the breaks in this case are technically not needed.  I think we should write our ` break ` statements as you have in switch statements.  The bottom line is that not all switch statements will ` return ` and in that case we need to remember to add a ` break `.  Its a good habit to get in to ( and easy to undo if needed );